ui/src/main/java/org/eclipse/starter/ui/Project.java

```diff
@@ -100,7 +285,6 @@ public void generate() {
 			File workingDirectory = Files.createTempDirectory("starter-output-").toFile();
 			LOGGER.log(Level.INFO, "Working directory: {0}", new Object[] { workingDirectory.getAbsolutePath() });
 
-			
 			LOGGER.info("Executing Maven Archetype.");
 			Properties properties = new Properties();
 			properties.putAll(Map.ofEntries(
```

Comment:
maybe not related to this commit, but can be rewritten as follows: 

```
			properties.putAll(Map.of(
					"jakartaVersion",
					((jakartaVersion % 1.0 != 0) ? String.format("%s", jakartaVersion)
							: String.format("%.0f", jakartaVersion)),
					"profile", profile,
					"javaVersion", javaVersion,
					"docker", (docker ? "yes" : "no"),
					"runtime", runtime
			));
````

Reply:
Again, we would be basically be at the 10 limit. If we add even one more thing like the group ID, artefact ID, etc, things would break.